README.md

```diff
@@ -1,55 +1,472 @@
+| ACTILITY_LIBRARY                                | Together with symbol STM_LIBRARY selects type firmware upgrade performed.                                  | In current project only ACTILITY_LIBRARY = 1 is supported                                                                                                                                                        |
+| STM_LIBRARY                                     | Together with symbol ACTILITY_LIBRARY selects type firmware upgrade performed.                             | In current project only STM_LIBRARY = 0 is supported                                                                                                                                                             |
+| INTEROP_TEST_MODE                               | Select interoperability test mode of the project. Data fragments are received in RAM if this mode selected | Default setting for the project INTEROP_TEST_MODE = 0 Setting it to 1 not recommended for normal operation                                                                                                       |
+| ACTIVE_REGION                                   | Together with defining REGION_xxx symbol selects ISM band                                                  | Could be set to: LORAMAC_REGION_AS923 LORAMAC_REGION_EU868 LORAMAC_REGION_US915                                                                                                                                  |
+| REGION_\<name\>                                 | Defines RF regions which are included during compilation, e.g. REGION_EU868                                | It is enough to define symbol in order to include support for correspondent RF region. It is not advised to select more than one region. Following regions are supported REGION_AS923 REGION_EU868 REGION_US915  |
+| STM32L476xx USE_HAL_DRIVER USE_STM32L4XX_NUCLEO | Internal project configuration                                                                             | Do not change this symbols settings or remove them !                                                                                                                                                             |
+
+*Device credentials configuration*
+
+In order to customize the devEUI/JoinEUI/AppKey/GenAppKey device parameters use
+following procedure:
+
+Edit
+`<root_dir>\Projects\STM32L476RG-Nucleo\Applications\LoRaWAN_Fuota1\Fuota\LoRaWAN\App\inc\Commissioning.h`
+
+-   Modify one of these parameters: `LORAWAN_DEVICE_EUI`, `LORAWAN_JOIN_EUI`,
+    `LORAWAN_APP_KEY`, `LORAWAN_GEN_APP_KEY`
+
+-   Recompile the firmware by executing the last step of section XXX
```

Comment:
We have to remove this line. It left from original version.

Reply:
I modified a bit the formatting/wording.
Do you want to completely remove this section ? Don't you think it can be useful to indicate how to customize the device identity?

Reply:
Not the whole section but just line advising to recompile because we are detailing build procedure in another section

Reply:
ok, done